README.md

```diff
@@ -1,284 +1,86 @@
-
-```
-GENESIS_TXN_PATH=network/genesis/local-genesis.txn TEST_AGENT_PUBLIC_DID_SEED=000000000000000000000000Trustee9 yarn test
-```
+<p align="center">
+  <a href="#features">Features</a> &nbsp;|&nbsp;
+  <a href="#getting-started">Getting started</a> &nbsp;|&nbsp;
+  <a href="#contributing">Contributing</a> &nbsp;|&nbsp;
+  <a href="#license">License</a> 
+</p>
 
-## Usage with Docker
+Aries Framework JavaScript is a typescript framework for building **SSI Agents and DIDComm services** that aims to be **compliant and interoperable** with the standards defined in the [Aries RFCs](https://github.com/hyperledger/aries-rfcs).
 
-If you don't want to install the libindy dependencies yourself, or want a clean environment when running the framework or tests you can use docker.
+## Features
 
-Make sure you followed the [local ledger setup](#setup-ledger) to setup a local indy pool inside docker.
+Some features are not yet supported, but are on our roadmap. Check [the roadmap](https://github.com/hyperledger/aries-framework-javascript/issues/39) for more information.
 
-```sh
-# Builds the framework docker image with all dependencies installed
-docker build -t aries-framework-javascript .
+- ✅ React Native
+- ✅ Node.JS
+- ✅ Issue Credential Protocol ([RFC 0036](https://github.com/hyperledger/aries-rfcs/blob/master/features/0036-issue-credential/README.md))
+- ✅ Present Proof Protocol ([RFC 0037](https://github.com/hyperledger/aries-rfcs/tree/master/features/0037-present-proof/README.md))
+- ✅ Connection Protocol ([RFC 0160](https://github.com/hyperledger/aries-rfcs/blob/master/features/0160-connection-protocol/README.md))
+- ✅ Basic Message Protocol ([RFC 0095](https://github.com/hyperledger/aries-rfcs/blob/master/features/0095-basic-message/README.md))
+- ✅ Indy Credentials (with `did:sov` support)
+- ✅ HTTP Transport
+- 🚧 Revocation of Indy Credentials
+- 🚧 Electron
+- 🚧 Mediator Coordination Protocol ([RFC 0211](https://github.com/hyperledger/aries-rfcs/blob/master/features/0211-route-coordination/README.md))
+- 🚧 WebSocket Transport
+- ❌ Browser
+- ❌ Connection-less Issuance and Verification
+- ❌ Issue Credential V2, Present Proof V2, DID Exchange Protocol, Out-Of-Band
+- ❌ W3C Linked Data VCs, BBS+ Signatures
 
-# Run tests without network
-docker run -it --rm aries-framework-javascript  yarn test -t "agents"
+## Getting Started
 
-# Run test with mediator agents and ledger pool
-docker-compose -f docker/docker-compose-mediators.yml up -d # Run alice-mediator and bob-mediator
-docker run --rm --network host --env TEST_AGENT_PUBLIC_DID_SEED=000000000000000000000000Trustee9 --env GENESIS_TXN_PATH=network/genesis/local-genesis.txn aries-framework-javascript yarn test
-```
+- NodeJS: [setup guide](./docs/setup-nodejs.md)
+- React Native: [setup guide](./docs/setup-react-native.md)
+- Electron: [setup guide](./docs/setup-electron.md)
```

Comment:
It would be good to get to Getting Started content directly from here. I suggest to take the content from docs/README.md and put it in here and remove docs/README.md file


## Getting Started

### Platform Specific Setup
...

### Usage
...

Or better word than Usage 😄 🤷‍♂️